SportsHub/SportsHub.Api/Controllers/UserController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SportsHub.Domain.Models;
+using SportsHub.Domain.Models.Constants;
+using System.Security.Claims;
+
+namespace SportsHub.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        [HttpGet("Public")]
+        public IActionResult Public()
+        {
+            return Ok("Public property");
+        }
+
+        [HttpGet("Admins")]
+        [Authorize]
```

Comment:
add [Authorize] attribute under the class. I don't think that unauthorized users suppose to have access on that class

Reply:
Is this meant for the Public class or AdminsEndpoint class, because AdminsEndpoint already has [Authorize]

Reply:
Implemented